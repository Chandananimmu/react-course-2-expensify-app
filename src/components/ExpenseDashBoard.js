import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilter from "./ExpenseListFilter";
const ExpenseDashBoard = () => (
  <div>
    <ExpenseList />
    <ExpenseListFilter />
  </div>
);
export default ExpenseDashBoard;
