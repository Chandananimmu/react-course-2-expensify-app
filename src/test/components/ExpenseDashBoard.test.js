import React from "react";
import { shallow } from "enzyme";
import ExpenseDashBoard from "../../components/ExpenseDashBoard";
test("shud render header correctly", () => {
  const wrapper = shallow(<ExpenseDashBoard />);
  expect(wrapper).toMatchSnapshot();
});
