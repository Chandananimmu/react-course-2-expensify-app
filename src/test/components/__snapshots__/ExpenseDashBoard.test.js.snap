// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shud render header correctly 1`] = `
<div>
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilter) />
</div>
`;
